refactor: split and sort words by js

zhuangya opened the pull request.

zhuangya committed:
refactor: split and sort words by js(w/ cache)

zhuangya committed:
implement split word

zhuangya committed:
chore: seem osascript js can not write to webpage

so the cache will not work, remove it.

thalesmello reviewed (approved):
Hello. In the last version of Chrome, oascript stopped working. I have yet to find an alternative approach to make it work.

Anyways, thank you for your contribution. I like it.

zhuangya commented:
last version of Chrome?? what's the version number

for me, Chrome 61.0.3163.100 stil working, the result below is from hackernew today:

(there could be a problem about escaping `\` anyway, let me know the command of osascript is working or not please 😸 )

```bash
# nos at zion in ~ [13:01:52]
→ osascript -e 'tell application "Google Chrome" to tell active tab of window 1 to execute javascript "[...new Set(document.body.innerText.match(/\\w+/g))].sort().join('\\n')"'
0,1,10,16,2,2017,2b918cf,3,44b4a2f,5,6,64,7bee92b,API,About,Add,Allow,Anyways,Assignees,Attach,Blog,Chrome,Code,Comment,Commits,Contact,Contributor,Conversation,Delete,Edit,Explore,Files,Fork,GitHub,Hello,Help,I,In,Inc,Insights,Issues,Labels,Learn,Markdown,Marketplace,Merged,Milestone,No,None,Notifications,Preview,Privacy,ProTip,Projects,Pull,Revert,Reviewers,Security,Shop,Sign,Skip,Status,Styling,Terms,This,Training,Unstar,Unsubscribe,Verified,View,Watch,Wiki,Write,You,a,added,ago,all,alternative,an,and,approach,approved,assigned,authored,be,because,better,branch,by,cache,can,changed,changes,chore,clipboard,closed,commented,comments,commit,commits,content,contribution,day,deleted,description,dragging,dropping,e1a63f4,edits,files,find,for,from,have,hours,implement,into,is,it,js,last,like,lines,maintainers,make,master,merged,milestone,more,not,notifications,oascript,of,one,or,osascript,out,participants,pasting,provided,re,receiving,refactor,repository,request,requests,safely,seem,selecting,set,some,sort,specific,split,stopped,successfully,supported,thalesmello,thank,the,them,these,thread,title,to,under,version,vim,w,webcomplete,webpage,with,word,words,work,working,write,yet,you,your,zhuangya
```
